javaee/ejb-mdb-amqp/src/main/java/org/wildfly/swarm/ts/javaee/ejb/mdb/amqp/QueueMdb.java

```diff
@@ -0,0 +1,33 @@
+package org.wildfly.swarm.ts.javaee.ejb.mdb.amqp;
+
+import javax.ejb.ActivationConfigProperty;
+import javax.ejb.MessageDriven;
+import javax.ejb.TransactionAttribute;
+import javax.ejb.TransactionAttributeType;
+import javax.inject.Inject;
+import javax.jms.JMSException;
+import javax.jms.Message;
+import javax.jms.MessageListener;
+import javax.jms.TextMessage;
+
+@MessageDriven(activationConfig = {
+        @ActivationConfigProperty(propertyName = "connectionFactory", propertyValue = "my-factory"),
+        @ActivationConfigProperty(propertyName = "user", propertyValue = "amq"),
+        @ActivationConfigProperty(propertyName = "password", propertyValue = "amq"),
+        @ActivationConfigProperty(propertyName = "destination", propertyValue = "my-queue"),
+        @ActivationConfigProperty(propertyName = "jndiParameters", propertyValue = "java.naming.factory.initial=org.apache.qpid.jms.jndi.JmsInitialContextFactory;connectionFactory.my-factory=amqp://${env.MESSAGING_SERVICE_HOST:localhost}:${env.MESSAGING_SERVICE_PORT:5672};queue.my-queue=my-queue"),
```

Comment:
This is more a question than a review, where this env property comes from?

Reply:
Hmm, good point. It doesn't, so the defaults always apply. Will remove the interpolation.

Reply:
Done.